content/W2015/W15-blog-hollyxu.md

```diff
@@ -0,0 +1,51 @@
+Title: Get to Know: Holly Xu
+Date: 2015-01-23 19:00
+Category: Blog
+Tags: get to know
+Slug: get-to-know-holly-xu
+Author: Julia Nguyen
+Summary: This week I sat down with Holly Xu, a fourth year student to talk about her transition into CS from accounting, how robotics club boosted her confidence, and the importance of independence.
+
+There's a recurring theme in these interviews - taking risks. Overcoming self-esteem is a huge part of being a woman in computing. This week I sat down with Holly Xu, a fourth year student to talk about her transition into CS from accounting, how robotics club boosted her confidence, and the importance of independence.
+
+![Photo of Holly Xu](http://i725.photobucket.com/albums/ww252/itsjulianguyen/holly_xu_zps74dlwd86.png "Holly Xu")
+
+Holly had a difficult decision to make when it came to applying to university. She had always been interested in computers, but she faced familial pressure to apply for accounting, an industry with a traditionally secure job market.
+
+In high school, she was a website and t-shirt designer for the robotics club. She had always declined offers from her teammates to learn how to solder and cut metal. Despite her experience in web development, Holly felt like she was not cut out for computer science because of her inexperience with writing algorithms. She believed her interests in web development was more of an artistic hobby.
+
+Despite receiving multiple offers to engineering programs, Holly settled on the Mathematics/Chartered Professional Accountancy program at the University of Waterloo. Between her first and second year, she realized she had no passion whatsoever for accounting. She envied her engineering friends who worked on exciting software projects.
```

Comment:
When Holly joined, it would have been known as "Mathematics/Chartered Accountancy"; the CPA change only happened recently. Not sure which would be better.